packages/select/src/common/listItemsProps.ts

```diff
@@ -46,6 +46,13 @@ export interface ListItemsProps<T> extends Props {
      */
     activeItem?: T | CreateNewItem | null;
 
+    /** If provided, this function will be used in place of the default implementation. */
```

Comment:
I borrowed the wording from https://github.com/palantir/blueprint/blob/b6c1518ea2f524accfba8045ce6f22cac7c5b56e/packages/core/src/context/hotkeys/hotkeysProvider.tsx#L98